### ISSUE TITLE
Login With Google

### ISSUE THREAD

**Issue by trey-rosius** [ConnectyCube/connectycube-flutter-samples#77]
**Specify the sample to which the issue belongs (use [x]):**
[x] Chat sample
[x] P2P Call sample
[ ] Conference Call sample

**Platform (use [x])**
[ ] Android
[ ] iOS
[x] Flutter

Hi, it's not clearly stated in the documents, how to authenticate with Google. I see only facebook and twitter. Or is there something I'm missing

**Comment by TatankaConCube** [comment]
You are right, in the current version of our API we don't have a separate auth provider for authentication with Google. You can implement it using common authentication via e-mail.
